### DISCUSSION TITLE
cypress-plugin-visual-regression-diff does not store actual snapshots if there is not difference between actual and previous snapshots

### DISCUSSION THREAD

**Discussion by parulbhagat-soprabanking** [FRSOURCE/cypress-plugin-visual-regression-diff#226] [Q&A]
Hi ,

The issue is that when we run the cypress tests, then only first image captured by cypress-plugin-visual-regression-diff plugin is stored at snapshots location and the actual image captured in the second run is temporarily made and then deleted once the tests are completed if there is not difference between actual and previous snapshot. Is there a way to store the actual and the difference image in the snapshots directly because our mochawesome reporter fails saying that the actual image is not found as it gets deleted after the run completes?

**Comment by FRSgit** [comment]
Hey! Changes landed in `4.0.0`. On top of that, I've added instructions on how to integrate this library with mochawesome to the [READMEs' FAQ section](https://github.com/FRSOURCE/cypress-plugin-visual-regression-diff#faq). Feel free to try it out!
